### PR TITLE
[rag eval] add claims counters

### PR DESCRIPTION
**Pull request by chejennifer**
update claim counter page to have 2 sections w/ 3 counters each:
1. statistical claims
    - total claims
    - false claims
    - unique tables
3. inferred claims
    - total claims
    - false claims
    - unsubstantiated claims

Screenshot 2024-06-24 at 3 07 33 PM

### CODE REVIEW

File: static/js/apps/eval_retrieval_generation/eval_list.tsx [hunk 1]
```diff
@@ -65,17 +69,17 @@ export function EvalList(): JSX.Element {
...
-                queryFeedbackResults[i][key] !== "0" &&
-                !queryFeedbackResults[i][key]
-              ) {
+              QUERY_TOTAL_STAT_CLAIMS_KEY,
+              QUERY_FALSE_STAT_CLAIMS_KEY,
+              QUERY_TOTAL_INF_CLAIMS_KEY,
+              QUERY_FALSE_INF_CLAIMS_KEY,
+              QUERY_UNSUB_INF_CLAIMS_KEY,
+              QUERY_TABLES_USED_KEY,
+            ].forEach((countKey) => {
```
Comment (shifucun) [inline static/js/apps/eval_retrieval_generation/eval_list.tsx:81]: make the list a constant even?

File: static/js/apps/eval_retrieval_generation/data_store.tsx [hunk 2]
```diff
@@ -162,22 +155,14 @@ export async function saveToSheet(
...
   callId: number,
-  response?: Response,
-  overallFeedback?: string,
-  totalClaims?: number,
-  falseClaims?: number
+  cellValues: Record<string, string | number>
 ): Promise<void> {
   const sheet = doc.sheetsByTitle[DC_FEEDBACK_SHEET];
   sheet.addRow({
+    ...cellValues,
```
Comment (shifucun) [inline static/js/apps/eval_retrieval_generation/data_store.tsx:162]: nice!

File: static/js/apps/eval_retrieval_generation/eval_list.tsx (hunk 1 above)
Comment (chejennifer) [reply static/js/apps/eval_retrieval_generation/eval_list.tsx]: done

File: static/js/apps/eval_retrieval_generation/rag_ans_feedback.tsx [hunk 3]
```diff
@@ -22,21 +22,42 @@ import { Button } from "reactstrap";
...
+  [QUERY_TABLES_USED_KEY]: 0,
+};
+const COUNTER_LABELS = {
+  [QUERY_TOTAL_STAT_CLAIMS_KEY]: "Total claims",
+  [QUERY_FALSE_STAT_CLAIMS_KEY]: "False claims",
+  [QUERY_TOTAL_INF_CLAIMS_KEY]: "Total claims",
+  [QUERY_FALSE_INF_CLAIMS_KEY]: "False claims",
+  [QUERY_UNSUB_INF_CLAIMS_KEY]: "Unsubstantiated claims",
+  [QUERY_TABLES_USED_KEY]: "Unique tables used",
+};
```
Comment (shifucun) [inline static/js/apps/eval_retrieval_generation/rag_ans_feedback.tsx:55]: oh, maybe put all of these together with the key list into one object? That's easier to track.

**Review by shifucun — APPROVED** [review]
nice clean up!

File: static/js/apps/eval_retrieval_generation/rag_ans_feedback.tsx (hunk 3 above)
Comment (chejennifer) [reply static/js/apps/eval_retrieval_generation/rag_ans_feedback.tsx]: good idea, updated

**Comment by chejennifer** [comment]
thanks for the review!

**Pull request merged**